File: src/Page/MyNotificationsPage.java
package Page;

import SystemMessage.FunctionType;
import SystemMessage.SystemMessage;

public class MyNotificationsPage extends Page{
	
	public MyNotificationsPage(String content) {
		super(content);
		setType(PageType.MY_NOTIFICATIONS_PAGE);
		setName("My Notifications Page");
	}
	
	@Override
	public SystemMessage runPage() {
		showContent();
		takeInput();
		return new SystemMessage(FunctionType.READ_NOTIFICATIONS, PageType.MAIN_MENU_PAGE, null);
	}
}

File: python_development/Page/MyNotificationsPage.py
from python_development.Page.Page import Page
from python_development.Page import PageType
from python_development.System import FunctionType
from python_development.System import SystemMessage


class MyNotificationsPage(Page):

    def __init__(self, content):
        super().__init__(content)
        self.setType(PageType.MY_NOTIFICATIONS_PAGE)
        self.setName("My Notifications Page")


    def runPage(self):
        self.showContent()
        self.takeInput()
        return SystemMessage(FunctionType.READ_NOTIFICATIONS, PageType.MAIN_MENU_PAGE, None)
    

    